### DISCUSSION TITLE
Does Harlequin run in Positron?

### DISCUSSION THREAD

**Discussion by daranzolin** [posit-dev/positron#4371] [Q&A] [answered]
Before I pester IT to come install Positron on my machine, I'd be very curious to learn if [Harlequin, a SQL IDE for terminals,](https://harlequin.sh/) runs smoothly in the Positron terminal [(it is not so smooth in RStudio).](https://github.com/rstudio/rstudio/issues/14745) Has anyone tried this?

**Comment by jmcphers** [comment, marked as answer]
Do you have access to VS Code? It'll work pretty much identically since VS Code and Positron use exactly the same terminal rendering engine (xterm.js). This is also the one RStudio uses, but RStudio uses an older version.

<img width="1262" alt="image" src="https://github.com/user-attachments/assets/b41c6b79-dfbe-4f4e-a174-1a8a5fe9aa03">

I'm not too familiar with Harlequin, but it looked pretty nice when I tried it!
